Analyse.cs

```diff
@@ -53,4 +59,20 @@ public Analyse(Input input)
             }
         }
     }
+
+    /// <summary>
+    /// Finds all words longer than 7 letters.
+    /// </summary>
+    /// <param name="input">The Input object to review.</param>
+    /// <returns>An array of words longer than 7 letters.</returns>
+    private List<string> GetLongWords(Input input)
+    {
+        List<string> longWords = 
+            (
+            from word in Regex.Matches(input.InputText, "[a-z']{8,}")
```

Comment:
Might it be worth using `[A-Za-z]` to also capture uppercase letters?

You could also use `\w` which will capture all "word" characters

Reply:
Good catch! I'll add an ignore case option.
Thanks.
```suggestion
            from word in Regex.Matches(input.InputText, "[\w']{8,}", RegexOptions.IgnoreCase)
```